crates/net/nat/src/lib.rs

```diff
@@ -48,6 +49,9 @@ pub enum NatResolver {
     PublicIp,
     /// Use the given [`IpAddr`]
     ExternalIp(IpAddr),
+    /// Resolve external IP via the network interface.
+    #[cfg(not(target_os = "windows"))]
```

Comment:
I'd like to not feature gate this variant

Reply:
it only works on windows

Reply:
https://docs.rs/if-addrs/0.13.3/if_addrs/fn.get_if_addrs.html

Reply:
https://docs.rs/if-addrs/0.13.3/x86_64-pc-windows-msvc/if_addrs/fn.get_if_addrs.html 

this also exists for windows,